kolibri/core/assets/test/vue.content-renderer.js

```diff
@@ -0,0 +1,505 @@
+/* eslint-env mocha */
```

Comment:
This header is required to stop our linter going mental over the syntax that mocha requires.